ibis/backends/duckdb/tests/test_udf.py

```diff
@@ -57,6 +57,19 @@ def test_builtin_scalar(con, func):
     assert con.execute(expr) == expected
 
 
+def test_builtin_scalar_noargs(con):
+    @udf.scalar.builtin
+    def version() -> str:
+        ...
+
+    expr = version()
```

Comment:
The error before was an expression level error. Would you mind adding a test like this in `ibis/tests/expr/test_udf.py` to test just the expression handling in a backend-agnostic way? Something like (untested):

```python
def test_builtin_scalar_noargs():
    @udf.scalar.builtin
    def version() -> str:
        ...

    expr = version()
    assert expr.type().is_integer()
    assert expr.op().shape is ibis.expr.datashape.Scalar
```

Reply:
done, but had to modify the last line to be
```python
 assert type(expr.op().shape) is ibis.expr.datashape.Scalar
 ```
 to get the test to pass though